src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.